Hero variants (#1324)
code-obos/grunnmuren: .changeset/thick-glasses-fall.md

    <Hero variant="two-column">
        <Content>
            <Heading level={1}>Dette er en Hero</Heading>
            <Description>– et samarbeidsprosjekt med Nordr</Description>
        </Content>
        <Media>
            <img
            src="https://res.cloudinary.com/obosit-prd-ch-clry/image/upload/f_auto,c_limit,w_2048,q_auto/v1582122753/Boligprosjekter/Oslo/Ulven/Ulven-N%C3%A6romr%C3%A5de-Oslo-OBOS-Construction-city.jpg"
            alt=""
            />
        </Media>
    </Hero>
```

### standard with `xl` heading
``` tsx
    // variant="standard" is default so that prop can be omitted, and the heading size set to `xl` on the `<Heading>`
    <Hero>
        <Content>
            <Heading level={1} size="xl">Dette er en Hero</Heading>
            <Description>– et samarbeidsprosjekt med Nordr</Description>
        </Content>
        <Media>
            <img
            src="https://res.cloudinary.com/obosit-prd-ch-clry/image/upload/f_auto,c_limit,w_2048,q_auto/v1582122753/Boligprosjekter/Oslo/Ulven/Ulven-N%C3%A6romr%C3%A5de-Oslo-OBOS-Construction-city.jpg"
            alt=""
            />
        </Media>
    </Hero>
```

### Heading

> To achieve this control of the heading size, I have added a new `size` prop to the `<Heading>` component. This way we can implement the same API for the Card component (which should support different heading sizes as well).

Author: ```suggestion
To achieve this control of the heading size a new `size` prop has been added to the `<Heading>` component. This way we can implement the same API for the Card component (which should support different heading sizes as well).
```
